modules/installation-vsphere-machines.adoc

```diff
@@ -86,7 +58,7 @@ $ base64 -w0 <installation_directory>/master.ign > <installation_directory>/mast
 +
 [source,terminal]
 ----
-$ base64 -w0 <installation_directory>/worker.ign > <installation_directory>/worker.64
+$ base64 -w0 <installation_directory>/compute.ign > <installation_directory>/compute.64
```

Comment:
Has the installer changed the names of the Ignition config files that it generates? The user doesn't name these.

Reply:
Ah, good point...no change from dev on these names so it should stay as `worker.ign`.